tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -60,13 +60,133 @@ class TestModelEndpointsOperations(TestMLRunSystem):
 
     def setup_method(self, method):
         super().setup_method(method)
-        if method.__name__ == "test_list_endpoints_without_creds":
+        if method.__name__ in [
+            "test_list_endpoints_without_creds",
+            "test_get_model_endpoint_metrics",
+        ]:
             return
         self.project.set_model_monitoring_credentials(
             stream_path=mlrun.mlconf.model_endpoint_monitoring.stream_connection,
             tsdb_connection=mlrun.mlconf.model_endpoint_monitoring.tsdb_connection,
         )
 
+    def _generate_event(
+        self,
+        endpoint_id,
+        result_name,
+        event_kind="result",
+        app_name="my_app",
+    ):
+        result_kind = 0
+        start_infer_time = datetime.isoformat(datetime(2024, 1, 1, tzinfo=timezone.utc))
+        end_infer_time = datetime.isoformat(
+            datetime(2024, 1, 1, second=1, tzinfo=timezone.utc)
+        )
+        result_status = 0
+        result_value = 123
+
+        data = {
+            "endpoint_id": endpoint_id,
+            "application_name": app_name,
+            "result_name": result_name,
+            "result_kind": result_kind,
+            "event_kind": event_kind,
+            "start_infer_time": start_infer_time,
+            "end_infer_time": end_infer_time,
+            "result_status": result_status,
+            # make sure we can write apostrophes (ML-7535)
+            "result_extra_data": """{}""",
+            "result_value": result_value,
+        }
+        return data
+
+    @pytest.mark.skipif(
+        "MLRUN_MODEL_ENDPOINT_MONITORING__TSDB_CONNECTION"
+        not in TestMLRunSystem._get_env_from_file(),
+        reason="MLRUN_MODEL_ENDPOINT_MONITORING__TSDB_CONNECTION not defined",
+    )
+    @pytest.mark.parametrize("use_v3io_tsdb_connector", [True, False])
+    def test_get_model_endpoint_metrics(self, use_v3io_tsdb_connector):
+        if use_v3io_tsdb_connector:
+            tsdb_connection_string = "v3io"
+        else:
+            tsdb_connection_string = os.environ[
+                "MLRUN_MODEL_ENDPOINT_MONITORING__TSDB_CONNECTION"
+            ]
```

Comment:
Pretty sure this should be
```suggestion
            tsdb_connection_string = mlconf.model_endpoint_monitoring.tsdb_connection
```